devel/push

```diff
@@ -19,7 +19,7 @@ BASE_BUILDER_IMAGE="nextstrain/base-builder"
 
 for tag in "$@"; do
 
-    if [[ $(docker image inspect --format "{{.RepoDigests}}" $BASE_IMAGE:$tag) != '[]' || $(docker image inspect --format "{{.RepoDigests}}" $BASE_BUILDER_IMAGE:$tag) != '[]' ]]; then
+    if [[ $tag != 'latest' && ($(docker image inspect --format "{{.RepoDigests}}" $BASE_IMAGE:$tag) != '[]' || $(docker image inspect --format "{{.RepoDigests}}" $BASE_BUILDER_IMAGE:$tag) != '[]') ]]; then
```

Comment:
Interestingly, [in the failing run](https://github.com/nextstrain/docker-base/runs/7984133243?check_suite_focus=true) it looks like the `latest` pushes succeeded and it's actually the `build-` tag that exited before push. So I'm not sure that this is the right change. Will need to do some local debugging to verify.

Reply:
I don't think this is the right change, though it may fix the symptom in our usage of `./devel/push`. What's happening, I believe, is that `./devel/push latest build-…` gets run, this check passes for `latest` tag, the images tagged `latest` are pushed, and then on the next iteration of this `for` loop this check fails for the `build-…` tag because it refers to the same `latest` image which was just pushed.

One possible fix, I think, would be to loop over all tags and make this assertion. Then loop again to push.

Reply:
I just checked locally and you're right, `RepoDigests` has a value if that local image has already been pushed regardless of tag. I'll update with your proposed fix.